backend/tests/services/knowledge/group_kb/test_group_kb_fixtures.py

```
@@ -0,0 +1,251 @@
+
+"""
+Shared fixtures for group knowledge base permission tests.
+"""
+
+import pytest
+from sqlalchemy.orm import Session
+
+from app.core.security import get_password_hash
+
+# Import all models to ensure they're registered with Base
+from app.models import *
```

Ruff (0.15.2)

[error] `from app.models import *` used; unable to detect undefined names

(F403)